packages/calcite-components/src/components/date-picker-month-header/date-picker-month-header.tsx

```diff
@@ -129,7 +129,7 @@ export class DatePickerMonthHeader {
     return (
       <Fragment>
         <a
-          aria-disabled={`${this.prevMonthDate.getMonth() === activeMonth}`}
+          aria-disabled={toAriaBoolean(this.prevMonthDate.getMonth() === activeMonth)}
```

Comment:
This should be handled in https://github.com/Esri/calcite-design-system/pull/8402#discussion_r1774508844., which is targeted for the September release. Can we roll this back to avoid merge conflicts? cc @anveshmekala